### ISSUE TITLE
Is there a good monorepo template?

### ISSUE THREAD

**Issue by thuringia** [bun-community/create-templates#17]
I know that some features for workspaces/monorepos are not yet supported but a template could still help,
especially with features like `bun build` and similar.

Does anyone have a template otherwise we could create one

**Comment by The-Code-Monkey** [comment]
Yeah this is a +1 from me, really need to see a way to setup a monorepo.
